controllers/pullRequestsController.js

```diff
@@ -0,0 +1,47 @@
+const logger = require('../utils/logger')
+const fetch = require('../lib/fetch')
+const config = require('../config')
+const { fetchUser } = require('../models/users')
+const { getNames } = require('../lib/getNames')
+
+/**
+ * Fetches the pull requests in Real-Dev-Squad by user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const pullRequests = async (req, res) => {
+  try {
+    const { user } = await fetchUser(req.params.id)
+    const url = `${config.baseUrl}/search/issues?q=org:${config.org}+author:${user.github_id}+type:pr`
```

Comment:
We should use check the rate limit rules and also pass the client secret and client Id.
https://developer.github.com/v3/search/#rate-limit

Reply:
Added authorization via personal access token
https://docs.github.com/en/free-pro-team@latest/rest/guides/getting-started-with-the-rest-api#authentication

They have not mentioned how to authorize using client id and client secret